### PR TITLE
CosmosDB 2023-09-15 release

### PR DESCRIPTION
**Pull request by rsharm7**
## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/CosmosDB/CosmosDB.Sdk/README.md [hunk 1]
```diff
@@ -27,16 +27,17 @@ title: CosmosDBManagementClient
 ### 
 ``` yaml 
 commit: main
```
Comment (VeryEarly) [inline src/CosmosDB/CosmosDB.Sdk/README.md:29]: please pick a commit instead of main

**Review by VeryEarly — APPROVED** [review]

**Review by VeryEarly — APPROVED** [review]

**Review by BethanyZhou — APPROVED** [review]

**Comment by pjohari-ms** [comment]
@VeryEarly @BethanyZhou is there any issue in merging this PR?
@rsharm7 mentioned that there are keys present in this PR. Can you point to them?

These keys (if present) point to a test account that we have, and is used for testing the API which gives these keys as a response. Should we remove the corresponding test completely if there's an issue keeping the keys? This API hasn't changed in a while and these keys have been present in the sessionRecords for a while now.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**